README.md

```diff
@@ -90,7 +90,7 @@ There are a convection name for some exported modules:
 
 `WPP.group.demoteParticipant` - Demote a participant from admin.
 
-`WPP.group.getGroupInfo` - Get group information.
+`WPP.group.getGroupInfoFromInviteCode` - Get group information from invite code.
```

Comment:
It also can handle links, see here:
https://github.com/wppconnect-team/wa-js/blob/87953a33e8a162ef33474045593960829ac47c3c/src/group/functions/getGroupInfoFromInviteCode.ts#L31

```suggestion
`WPP.group.getGroupInfoFromInviteCode` - Get group information from an invitation link or an invite code.
```